src/AlgebraicGeometry/ToricVarieties/NormalToricVarieties/constructors.jl

```diff
@@ -178,6 +178,61 @@ end
 
 
 
+######################
+# Equality
+######################
+
+@doc raw"""
+    Base.:(==)(tv1::NormalToricVariety, tv2::NormalToricVariety)
+
+In OSCAR, the `==` operator is reserved to check if two normal toric varieties are identical,
+meaning their underlying polyhedral fans are the same. However, this check is computationally
+challenging due to several reasons:
+
+- The ray generators might be scaled.
+- The ray generators might be stored in different orders.
+- The maximal (polyhedral) cones of the polyhedral fan might be stored in different orders.
+- If we fall back on polyhedral fan equality, lineality of the cones must also be considered.
+
+To avoid this computational bottleneck, we have designed OSCAR to *not* implement the `==`
+method for toric varieties. Instead, using `==` will always raise an error, as shown below.
+
+However, triple equality `===` is supported for normal toric varieties. This returns `true`
+if the memory locations of the two objects in question are identical, and `false` otherwise.
+
+# Examples
+```jldoctest
+julia> p2 = projective_space(NormalToricVariety, 2)
+Normal toric variety
+
+julia> f2 = hirzebruch_surface(NormalToricVariety, 2)
+Normal toric variety
+
+julia> p2 === f2
+false
+
+julia> p2 === p2
+true
+
+julia> function capture_error_message()
+         try
+           p2 == f2
+         catch e
+           println(e.msg)
+         end
+       end
+capture_error_message (generic function with 1 method)
```

Comment:
This function is not needed. You can just let the doctest print the error, and truncate them with `[...]` once line numbers etc occurr